webpack/devserver.js

```diff
@@ -26,7 +26,7 @@ module.exports = function(webpackConfig) {
         chunks: false,
         chunkModules: false
       }
-    }).listen(8888, 'localhost', function(err) {
+    }).listen(8888, '0.0.0.0', function(err) {
```

Comment:
Yes, just a link why 0.0.0.0 is used. As u know, in Este.js we are trying to educate developers through little comments :-)